StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Embedded/EmbeddedPaymentElementConfiguration.swift

```diff
@@ -141,6 +141,15 @@ extension EmbeddedPaymentElement {
 
             /// The button says “Continue”. When tapped, the form sheet closes.
             case `continue`
+
+            var analyticValue: String {
+                switch self {
+                case .confirm:
+                    return "confirm"
+                case .`continue`:
+                    return "continue"
+                }
+            }
```

Comment:
Ah sorry, could we move this to `STPAnalyticsClient+PaymentSheet` too?

Reply:
oops, my bad, thanks!